state/futures/types.ts

```diff
@@ -59,6 +60,8 @@ export type FuturesQueryStatuses = {
 	positionHistory: QueryStatus;
 	trades: QueryStatus;
 	selectedTraderPositionHistory: QueryStatus;
+	isolatedMarginTransfers: QueryStatus;
```

Comment:
can just have a single query status here to match the one action `marginTransfers`